src/tests/ftest/soak/soak.py

```diff
@@ -226,25 +226,30 @@ def get_remote_logs(self):
         # copy the files from the remote
         # TO-DO: change scp
         this_host = socket.gethostname()
-        command = "/usr/bin/bash -c \"/usr/bin/rsync -avtr --min-size=1B " \
```

Comment:
Did Bandit actually flag this use of `/usr/bin/bash`?  If not, and it was just found by manual inspection, I wonder how we can automatically catch this in the future.  Perhaps `slurm_utils.srun` should fail if it finds a shell in it's `command` argument?

Reply:
Bandit checking was enabled when this was included and I do not believe it was called out.

Reply:
Probably because of two things:
1. Bandit probably knows nothing about slurm and
1. https://github.com/PyCQA/bandit/issues/609